ecl/hqlcpp/hqlcppsys.ecl

```diff
@@ -790,6 +790,7 @@ const char * cppSystemText[]  = {
     "   boolean newMemorySpillSplitArg(unsigned4 usageCount, const varstring name, boolean meta) : include, pseudoentrypoint='new CLibraryMemorySpillSplitArg';",
     "   boolean newWorkUnitReadArg(const varstring _name, boolean _meta) : include, pseudoentrypoint='new CLibraryWorkUnitReadArg';",
     "   boolean newWorkUnitWriteArg(const varstring _name, unsigned4 _flags, boolean _meta) : include, pseudoentrypoint='new CLibraryWorkUnitWriteArg';",
+    "   CThorExternalArg(unsigned4 _numInputs) : include;",
```

Comment:
I don't understand this line

Reply:
nor do I. Is it for a forward declaration?

Reply:
It is the declaration of the constructor for the helper class - so it can be called as a fake external function.

Reply:
Perhaps a comment?